src/Chainweb/SPV.hs

```diff
@@ -212,3 +209,8 @@ instance FromJSON (TransactionOutputProof SHA512t_256) where
 --
 outputProofChainId :: Getter (TransactionOutputProof a) ChainId
 outputProofChainId = to (\(TransactionOutputProof cid _) -> cid)
+
+-- A reasonable block history window for an SPV header oracle
+-- to look back over.
+defaultSpvProofExpirationWindow :: Word64
+defaultSpvProofExpirationWindow = 20_000
```

Comment:
I'd like this moved out even further, maybe even to its own file if necessary, as the minimum header oracle size.